src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -219,6 +219,24 @@ public CheckOutResponseDTO checkOutItem(InnReachTransaction transaction, UUID se
     return circulationClient.checkOutByBarcode(checkOut);
   }
 
+  @Override
+  public void createRecallRequest(UUID recallUserId, UUID itemId) {
+    var request = RequestDTO.builder()
+      .itemId(itemId)
+      .requesterId(itemId)
+      .requestType(RECALL.getName())
+      .requestDate(OffsetDateTime.now())
+      .fulfilmentPreference(HOLD_SHELF.getName())
+      .build();
```

Comment:
Looks like `pickupServicePointId` should be set
> using the recall user's default pickup location

Or just check if it implicitly uses default location when `pickupServicePointId` is not specified